lib/mongodb.js

```diff
@@ -802,6 +828,28 @@ MongoDB.prototype.count = function count(model, where, options, callback) {
 };
 
 /**
+ * Replace properties for the model instance data
+ * @param {String} model The model name
+ * @param {Object} data The model data
+ * @param {Function} [callback] The callback function
+ */
+MongoDB.prototype.replaceAttributes = function replaceAttrs(model, id, data, options, cb) {
+  var self = this;
+  if (self.debug) {
+    debug('replaceAttributes', model, id, data);
+  }
+  var oid = self.coerceId(model, id);
+  this.execute(model, 'update', {_id: oid}, data, {upsert: false}, function (err, info) {
+    if (err)
+      return cb && cb(err);
+    if (self.debug)
+      debug('replaceAttributes.callback', model, {_id: oid}, data, err, info);
```

Comment:
`err` is always falsy here. I think it's better to move logging on L504-505 above the error check on L502-503?